Services/DataBaseService/Repositories/UserRepository.cs

```diff
@@ -19,8 +19,10 @@ public UserRepository(IUserMapper mapper, TPlatformDbContext dbContext)
             this.dbContext = dbContext;
         }
 
-        public void CreateUser(User user)
+        public void CreateUser(User user, string email)
         {
+            if (dbContext.UsersCredentials.Any(userCredential => userCredential.Email == email))

```

Comment:
I think we should do a check that ignores capital letters

Reply:
Why?

Reply:
Isn't email Hey@there.com and hey@there.com are different?

Reply:
No, it`s same emails. And you should avoid it.

Reply:
In our db this is different email, but not in other world))

Reply:
Hmmm, Okay, I checked in some places and you are right. Thanks :)

Reply:
This request is translated to sql, and sql doesn't compare case of letter. So that would be checked automatically

Reply:
Hmmm, okey